gsl_core/templatetags/gsl_filters.py

```diff
@@ -43,16 +44,15 @@ def remove_first_word(value):
 
 
 @register.filter
-def create_alert_data(status: str | None, arg: str) -> dict[str, str | bool]:
-    data_dict: dict[str, str | bool] = {"is_collapsible": True}
-    if status is None:
-        data_dict["title"] = arg
-        return data_dict
-
-    data_dict["description"] = arg
-
-    if status in STATUS_TO_ALERT_TITLE:
-        data_dict["title"] = STATUS_TO_ALERT_TITLE[status]
+def create_alert_data(message: Any) -> dict[str, str | bool]:
+    data_dict: dict[str, str | bool] = {}
+    data_dict["is_collapsible"] = True
+    data_dict["description"] = message.message
```

Comment:
A-t-on besoin du typage ici ? (On l'a déjà ligne 47 dans le typehint du retour de fonction)
Ainsi qu'une suggestion pour que ce soit un peu moins verbeux.

```suggestion
    data_dict = {
        "is_collapsible": True,
        "description": message.message,
    }
```

Reply:
En tout cas c'est beaucoup plus lisible comme cela, bien joué.